src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -30,7 +30,7 @@ public void testCallCheck() {
 	@Test
 	public void testIsFull() {
 		DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
-		final int STACK_CAPACITY = 12;
+		final int STACK_CAPACITY = 11;
```

Comment:
Is it a correct approach to fixing errors showed by tests?
Errors are still within the project. STACK_CAPACITY is 12 so 12 elements should be added to the stack.